src/vs/platform/terminal/node/ptyService.ts

```diff
@@ -29,10 +35,20 @@ export class PtyService extends Disposable implements IPtyService {
 	readonly onProcessOverrideDimensions = this._onProcessOverrideDimensions.event;
 	private readonly _onProcessResolvedShellLaunchConfig = this._register(new Emitter<{ id: number, event: IShellLaunchConfig }>());
 	readonly onProcessResolvedShellLaunchConfig = this._onProcessResolvedShellLaunchConfig.event;
-
-	constructor(
-	) {
+	private _logService: ILogService;
+	private _instantiationService: InstantiationService;
+	constructor() {
 		super();
+		this._instantiationService = new InstantiationService();
+		this._logService = this._instantiationService.createInstance(ILogService);
```

Comment:
An `ILogService` should come in when you remerge tyriar/megan, if we need an `IInstantiationService` that would go into ptyHostMain.ts but I think we can avoid that.

Reply:
there are errors if I remove that